### ISSUE TITLE
Deviantart Module Doesn't DL Full Content

### ISSUE THREAD

**Issue by g-l-i-t-c-h-o-r-s-e** [Bionus/imgbrd-grabber#2917]
If you try to save the full image or any other downloadable content (like a .swf) that's hosted on deviantart, 
Grabber only downloads the thumbnail for the post; even if I'm logged in to deviantart in Grabber.
Ideally it would be far more desirable to have the ability to download the "Free Download" content if you're signed in on Grabber, so that one can scrape higher resolution content and more :D I'm trying to collect a lot of .swf's for example, and this would remedy that
![image](https://user-images.githubusercontent.com/17163949/221358049-04a22768-85df-4566-9cc4-2f0f76c7dcdb.png)

**Comment by Bionus** [comment]
Indeed, I never really checked out other things than images on DA so I never noticed this button. Just pushed a fix 👍

**Comment by Bionus** [comment]
Good catch, just pushed a fix for that as well 👍 
This case was handled but seems like there was a small bug in there 😅

**Comment by Railline** [comment]
Hey i'm sorry to reopen this tread but i still have the same problem in 7.12.2

**Comment by Railline** [comment]
i have found that, maybe this is the cause of the poor resolution download
https://gist.github.com/micycle1/735006a338e4bea1a9c06377610886e7#file-a-md
